features/user_sign_in.feature

```diff
@@ -0,0 +1,14 @@
+Feature: User sign in
+
+  As a Viewer, Publisher or Admin
+
+  Background:
+    Given I am not signed in
+
+  Scenario: A user signing in with valid credentials
```

Comment:
Still getting to grips with feature tests, should we be testing other cases, like when a user attempts to sign in but has not provided an email or password and that the screen shows the validation errors?